solr-search/src/main/java/edu/unc/lib/dl/search/solr/model/HierarchicalBrowseResultResponse.java

```diff
@@ -86,11 +87,12 @@ public void removeContainersWithoutContents() {
         ListIterator<BriefObjectMetadata> resultIt = this.getResultList().listIterator(this.getResultList().size());
         while (resultIt.hasPrevious()) {
             BriefObjectMetadata briefObject = resultIt.previous();
-            if (briefObject == null || briefObject.getContentModel() == null) {
+            if (briefObject == null || briefObject.getResourceType() == null) {
                 continue;
             }
+            String resourceType = briefObject.getResourceType();
             if ((!briefObject.getCountMap().containsKey("child") || briefObject.getCountMap().get("child") == 0)
-                    && briefObject.getContentModel().contains(ContentModelHelper.Model.CONTAINER.toString())) {
+                    && !ResourceType.File.equals(resourceType)) {
```

Comment:
why was this part of the condition changed to check for the type 'File', when before it checked for 'CONTAINER'?

Reply:
ContentModel is a fedora3 concept, so was removing that. It was changed from checking that a thing was a container to checking that a thing was NOT a file, which are roughly the same for this purpose.